github-pullrequest-plugin/src/main/resources/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger/help.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    Trigger that run job for GitHub Pull Request activities.
+    Trigger that launches builds in response to GitHub Pull Request activities.
```

Comment:
Maybe copy event name and it's description from GH UI? https://github.com/jenkinsci/github-plugin/pull/54#issuecomment-123095439